sdks/java/io/cassandra/src/test/resources/cassandra.yaml

```diff
@@ -0,0 +1,1073 @@
+#
```

Comment:
Clean up this file - remove cruft, unnecessary settings and unnecessary comments?